Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -21,6 +21,7 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;

```

Comment:
This package has a strict no Linq policy as Linq has proven to add a lot of size to builds and can be quite slower.
Details on usage below.